packages/0xcert-ethereum-asset-ledger/src/mutations/revoke-abilities.ts

```diff
@@ -2,20 +2,21 @@ import { Mutation } from '@0xcert/ethereum-generic-provider';
 import { encodeParameters } from '@0xcert/ethereum-utils';
 import { AssetLedger } from '../core/ledger';
 
-const functionSignature = '0xf394b6df';
-const inputTypes = ['address', 'uint256'];
+const functionSignature = '0xaca910e7';
+const inputTypes = ['address', 'uint256', 'bool'];
 
 /**
  * Revokes(removes) abilities from account.
  * @param ledger Asset ledger instance.
  * @param accountId Address of the account for which abilities will be revoked.
  * @param abilities Abilities as number representing bitfield.
+ * @param allowSuperRevoke Additional check so that you do not remove your own super ability by mistake.
```

Comment:
`Additional check that prevents you from removing your own super ability by mistake.`